### ISSUE TITLE
wrist.osim model does not open

### ISSUE THREAD

**Issue by carmichaelong** [opensim-org/opensim-gui#1351]
@aymanhab 

Model wrist.osim provided with installer opens in 4.3 without any problems. Does not open with the 4.4 beta GUI.

```
java.lang.IllegalArgumentException: Illegal Capacity: -1
	at java.util.ArrayList.<init>(ArrayList.java:157)
	at org.opensim.threejs.ModelVisualizationJson.createJsonForGeometryPath(ModelVisualizationJson.java:1704)
	at org.opensim.threejs.ModelVisualizationJson.processDecorationsForComponent(ModelVisualizationJson.java:509)
	at org.opensim.threejs.ModelVisualizationJson.createJsonForModel(ModelVisualizationJson.java:467)
	at org.opensim.threejs.ModelVisualizationJson.<init>(ModelVisualizationJson.java:427)
	at org.opensim.view.pub.ViewDB.getJsonForModel(ViewDB.java:1147)
	at org.opensim.view.pub.ViewDB.update(ViewDB.java:401)
	at java.util.Observable.notifyObservers(Observable.java:159)
	at org.opensim.view.pub.OpenSimDB.addModel(OpenSimDB.java:153)
	at org.opensim.view.pub.OpenSimDB.addModel(OpenSimDB.java:141)
	at org.opensim.view.FileOpenOsimModelAction$2.run(FileOpenOsimModelAction.java:100)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:758)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:74)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:728)
	at org.netbeans.core.TimableEventQueue.dispatchEvent(TimableEventQueue.java:136)
[catch] at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:205)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

**Comment by aymanhab** [comment]
Thanks for the find @carmichaelong I'm looking into it now

**Comment by aymanhab** [comment]
Fix submitted in PR #1352 Please test when artifacts are ready. Thanks much for the find @carmichaelong

**Comment by aymanhab** [comment]
@carmichaelong I think this was fixed, if confirmed please close. Thank you
